bundles/org.openhab.binding.insteon/src/main/java/org/openhab/binding/insteon/internal/message/DataType.java

```diff
@@ -31,10 +31,10 @@
     ADDRESS("address", 3),
     INVALID("INVALID", -1);
 
-    private static HashMap<String, DataType> typeMap = new HashMap<>();
+    private static HashMap<String, @Nullable DataType> typeMap = new HashMap<>();
```

Comment:
```suggestion
    private static Map<String, DataType> typeMap = new HashMap<>();
```

With the current EEAs there is only proper null analysis for the `Map` type. I created https://github.com/lastnpe/eclipse-null-eea-augments/pull/109 because I was not happy to see the analysis missing on other common types. So that should also bring proper analysis whenever someone doesn't program against interfaces such as `Map` (which is still preferable) but actual implementations such as `HashMap`, `ArrayList` etc.

Reply:
I made the change but I'm seeing the following in the maven build:

`[INFO] /home/rob/openhab/openhab-addons/bundles/org.openhab.binding.insteon/src/main/java/org/openhab/binding/insteon/internal/message/DataType.java:[77,16] Unsafe interpretation of method return type as '@NonNull' based on the receiver type 'java.util.@NonNull HashMap<java.lang.@NonNull String,org.openhab.binding.insteon.internal.message.@NonNull DataType>'. Type 'java.util.HashMap<K,V>' doesn't seem to be designed with null type annotations in mind`

Reply:
fixed